src/main/java/com/mathworks/ci/MatlabRunTestsStepExecution.java

```diff
@@ -58,10 +58,18 @@ private synchronized int execMatlabCommand(FilePath workspace, Launcher launcher
             TaskListener listener, EnvVars envVars) throws IOException, InterruptedException {
         final String uniqueTmpFldrName = getUniqueNameForRunnerFile();  
         try {
-            ProcStarter matlabLauncher = getProcessToRunMatlabCommand(workspace, launcher, listener, envVars,
-                    envVars.expand(getCommand()), uniqueTmpFldrName);
+            FilePath genScriptLocation =
+                    getFilePathForUniqueFolder(launcher, uniqueTmpFldrName, workspace);
+            final String cmdPrefix = "addpath(genpath('" + genScriptLocation.getRemote() + "')); ";
```

Comment:
Don't forget any time you're injecting a string value into MATLAB code, you have to escape single quotes.

Reply:
Thats good point :P . done